src/apps/LatexRender.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useRef, useState } from "react";
+import { InstantApp, InstantAppProps, queryToDataInput } from "./types";
+import { BlockMath } from 'react-katex';
+import styled from "styled-components";
+import 'katex/dist/katex.min.css';
+import html2canvas from "html2canvas";
+import jsPDF from "jspdf";
+
+const Container = styled.div`
+`;
+
+const InputField = styled.textarea`
+    width:100%;
+    border:none;
+    border-bottom:1px solid black;
+
+    &:focus{
+        border:none;
+    }
+`;
+
+const Alert = styled.div`
+`;
+
+const Button = styled.button`
+    border: none;
+    background: none;
+    border: 1px solid black;
+    padding: 6px 20px;
+    border-radius: 52px;
+    cursor:pointer;
+
+    &:active{
+        background:black;
+        color:white;
+    }
+`;
+
+
+const LatexRendererComponent = (props: InstantAppProps) => {
+    const [content, setContent] = useState("c = \\pm \\sqrt{a^2 + b^2}")
+    const captureInput = (e) => {
+        setContent(e.target.value)
+    }
+    const downloadLatex = (e) => {
+        const filename = 'latex-renderer-felvin-search.pdf';
+        html2canvas(document.querySelector('#rendererElement'),
+            { scale: 5 }
+        ).then(canvas => {
+            let pdf = new jsPDF();
+            pdf.addImage(canvas.toDataURL('image/png'), 'PNG', 0, 10, 215, 29);
+            pdf.save(filename);
```

Comment:
I don't know much about how this `jsPDF` function works, but the text inside the downloaded PDF seems very squished to me. What do you think? 
![image](https://user-images.githubusercontent.com/60931421/132120059-d3df83b3-99c4-4d42-95f7-fb7d7b2e0e3c.png)

Reply:
Actually huge part of my time got consumed on this, I'm trying to convert it to pdf but, there is no direct way to convert html element with css to pdf, here I'm converting it into a image (base64 string) and then inserting it into the pdf at position.
Providing correct dimension is somewhat difficult. I'm trying to solve it.

Reply:
Thank you for going the extra effort to add this, I will look into this as well and try to help fix this.

Reply:
Hi solved this 🥳, found a npm package that can convert html elements directly into pdf, it performs same actions under the hood